diff --git a/configure.ac b/configure.ac
index cc8686037d8fd12150106396d054d7411db21024..d876a2fd5e21c9649230c92623233063dab137dc 100644
--- a/configure.ac
+++ b/configure.ac
@@ -10,7 +10,7 @@ AC_INIT([dbus],[dbus_version],[https://bugs.freedesktop.org/enter_bug.cgi?produc
 
 AC_CONFIG_AUX_DIR([build-aux])
 
-m4_pattern_forbid([^AX_],
+m4_pattern_forbid([^AX_(CHECK_ENABLE_DEBUG|CODE_COVERAGE|COMPILER_FLAGS|COMPILER_FLAGS_(CFLAGS|CXXFLAGS|LDFLAGS))\b],
   [Unexpanded AX_ macro found. Please install GNU autoconf-archive])
 
 AC_CANONICAL_HOST
