--- dbus-1.6.19~20131112/doc/Makefile.am.nodoc~	2013-11-15 01:35:11.084661610 +0100
+++ dbus-1.6.19~20131112/doc/Makefile.am	2013-11-15 01:34:53.567759599 +0100
@@ -64,13 +64,13 @@ dist_html_DATA += $(STATIC_HTML)
 # we distribute these in the tarball so users don't necessarily need xmlto
 dist_html_DATA += $(XMLTO_OUTPUT)
 
+if DBUS_XML_DOCS_ENABLED
 XMLTO_OUTPUT=					\
 	dbus-faq.html				\
 	dbus-specification.html			\
 	dbus-test-plan.html			\
 	dbus-tutorial.html
 
-if DBUS_XML_DOCS_ENABLED
 dbus-specification.html: dbus-specification.xml
 	$(XMLTO) html-nochunks $<
 
