diff -Naur dbus-1.10.0/doc/Makefile.am dbus-1.10.0.tpg/doc/Makefile.am
--- dbus-1.10.0/doc/Makefile.am	2015-03-02 14:41:01.000000000 +0000
+++ dbus-1.10.0.tpg/doc/Makefile.am	2015-09-30 17:21:22.272146973 +0000
@@ -71,6 +71,7 @@
 	dbus-api-design.html \
 	$(NULL)
 
+if DBUS_XML_DOCS_ENABLED
 XMLTO_HTML =					\
 	dbus-faq.html				\
 	dbus-specification.html			\
@@ -79,7 +80,6 @@
 	$(MAN_HTML_FILES)			\
 	$(NULL)
 
-if DBUS_XML_DOCS_ENABLED
 html_DATA += $(XMLTO_HTML)
 
 %.html: %.xml
