diff -up dbus-1.13.18/bus/main.c.4~ dbus-1.13.18/bus/main.c
--- dbus-1.13.18/bus/main.c.4~	2020-06-10 10:46:48.000000000 +0200
+++ dbus-1.13.18/bus/main.c	2020-08-11 03:20:36.967698616 +0200
@@ -29,6 +29,7 @@
 #include <stdio.h>
 #include <stdlib.h>
 #include <string.h>
+#include <malloc.h>
 #ifdef HAVE_SIGNAL_H
 #include <signal.h>
 #endif
@@ -722,6 +723,13 @@ main (int argc, char **argv)
 #endif /* DBUS_UNIX */
 
   _dbus_verbose ("We are on D-Bus...\n");
+  /*
+   * during init we may have allocated and then freed memory, which
+   * the C library will buffer for us. This is a good place to really
+   * free that memory. Any memory we allocate/free during our normal
+   * operation will still use the C library buffers.
+   */
+  malloc_trim(0);
   _dbus_daemon_report_ready ();
   _dbus_loop_run (bus_context_get_loop (context));
 
